src/main/java/org/jvnet/hudson/test/SmokeTest.java

```diff
@@ -0,0 +1,7 @@
+package org.jvnet.hudson.test;
+
+/**
+ * Interface used as a Category for JUnit tests
```

Comment:
It would be nice to have a more specific description of what it does. It would be reflected in Javadoc